Lib/test/test_ipaddress.py

```diff
@@ -13,6 +13,7 @@
 import ipaddress
 import weakref
 from test.support import LARGEST, SMALLEST
+from collections.abc import Iterator
```

Comment:
```suggestion
from collections.abc import Iterator
from test.support import LARGEST, SMALLEST
```

(well, the imports are already unsorted  but let's try to add them in a sorted manner as much as possible)